src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/imports/importorder/InputImportOrderEclipseStaticRepetition.java

```diff
@@ -1,8 +1,21 @@
 //non-compiled with javac: contains specially crafted set of imports for testing
 package com.puppycrawl.tools.checkstyle.checks.imports.importorder;
-import static io.netty.handler.codec.http.HttpHeaders.Names.DATE;
```

Comment:
keep as it was, your comment should be over class.

Reply:
@romani ok I will correct all these.

Reply:
done